MyHeartCounts/MyHeartCountsStandard+HealthKit.swift

```diff
@@ -87,40 +78,77 @@ extension MyHeartCountsStandard: HealthKitConstraint {
             if let willUploadNotificationId {
                 UNUserNotificationCenter.current().removeDeliveredNotifications(withIdentifiers: [willUploadNotificationId])
             }
-            await showDebugHealthKitEventNotification(for: .deletedSamples(sampleType, Array(deletedObjects)), stage: .didUpload)
+            await showDebugHealthKitEventNotification(
+                for: .deleted(sampleTypeTitle: sampleType.displayTitle, count: deletedObjects.count),
+                stage: .didUpload
+            )
+        }
+    }
+}
+
+
+extension MyHeartCountsStandard {
+    func uploadHealthObservation(_ observation: some HealthObservation & Sendable) async throws {
+        try await uploadHealthObservations(CollectionOfOne(observation), batchSize: 1)
+    }
+    
+    func uploadHealthObservations(_ observations: some Collection<some HealthObservation & Sendable>, batchSize: Int = 100) async throws {
```

Comment:
We should eventually move this into the Template App; would be good if we create an issue for this in the template app. Not sure if it should be abstracted in an other package.